samples/matlab_experimental/plug_flow_reactor.m

```diff
@@ -8,19 +8,19 @@
 %
 % The reactor assumes that the flow follows the Ideal Gas Law.
 %
-% The governing equations used in this code can be referenced at:
+% The governing equations used in this code (see :ref:`PFR_solver.m
+% <sphx_glr_examples_matlab_experimental_PFR_solver.m>`) can be referenced at:
 % *S.R Turns, An Introduction to Combustion - Concepts and Applications,
 % McGraw Hill Education, India, 2012, 206-210.*
 %
 % The current example is written for methane combustion, but can be readily
 % adapted for other chemistries.
 %
-% Developed by Ashwin Kumar/Dr.Joseph Meadows (mgak@vt.edu/jwm84@vt.edu) on 3-June-2020
-% Research Assistant/Assistant Professor
-% Advanced Propulsion and Power Laboratory
-% Virginia Tech
+% Example originally developed by Ashwin Kumar (Research Assistant, mgak@vt.edu) and
```

Comment:
This (pre-existing) acknowledgment is out of the norm, as we usually add contributors elsewher.